content/docs/2.8/operate/cluster.md

```diff
@@ -120,3 +120,29 @@ To specify values other than their defaults, you can set the following environme
 | KEDA_METRICS_LEADER_ELECTION_LEASE_DURATION  | Metrics Server | LeaseDuration    |
 | KEDA_METRICS_LEADER_ELECTION_RENEW_DEADLINE  | Metrics Server | RenewDeadline    |
 | KEDA_METRICS_LEADER_ELECTION_RETRY_PERIOD    | Metrics Server | RetryPeriod      |
+
+## Restrict Secret Access
```

Comment:
It looks like this requires code changes which will only be in 2.9; would you mind removing changes here please?

Reply:
Sure, will remove it